models/user.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose')
+
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
+    username: {
+        type: String,
+        required: true,
+    },
+    email:{
+        type: String,
+        required: true,
```

Comment:
make it unique

Reply:
Cool, though I'm already checking if the email is used before in my resolver function.